scripts/install/install_unix.sh

```diff
@@ -354,23 +358,25 @@ set_package_type()
 
 # This will set the urls to use when downloading the collector and its plugins.
 # These urls are constructed based on the --version flag or COLLECTOR_VERSION env variable.
-# If not specified, the version defaults to "latest".
+# If not specified, the version defaults to whatever the latest release on github is.
 set_download_urls()
 {
-  if [ -z "$version" ] ; then
-    # shellcheck disable=SC2153
-    version=$COLLECTOR_VERSION
-  fi
-
   if [ -z "$url" ] ; then
-    url=$DOWNLOAD_BASE
+    if [ -z "$version" ] ; then
+      # shellcheck disable=SC2153
+      version=$COLLECTOR_VERSION
+    fi
 
-    # Only if we're using the base download url apply version formatting
     if [ -z "$version" ] ; then
```

Comment:
@cpheps is this the same check two times in a row with different results?

Reply:
ignore me. i get it now 😄

Reply:
Yeah it's mostly if we don't know the collector version the package it downloads too isn't named weird.